README.md

```diff
@@ -1,4 +1,12 @@
-# Venox Bot
-A bot used by [@srnyx](https://github.com/srnyx) (https://srnyx.xyz/) and [@ChrizxzFTW](https://github.com/Chrizxz) (https://chrizftw.cf/) in their administration.
+# venox-rewrite
 
-[![Website](https://img.srnyx.xyz/r/banner_round.png)](https://v.srnyx.xyz)
+<hr>
+
+Steps on how to start bot:
+1. Install `pie` version from [PIE-RUST](https://github.com/skandabhairava/pie-rust/releases/tag/main)
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/Venox-Network/venox/pullRequest?prid=9471648)
```suggestion

1. Install `pie` version from [PIE-RUST](https://github.com/skandabhairava/pie-rust/releases/tag/main)
```